CoEpi/domain/logic/CenMatcher.swift

```diff
@@ -42,8 +44,12 @@ class CenMatcherImpl: CenMatcher {
             possibleCENs.append(cen.toHex()) // No fixed size array
         }
 
-        os_log("Generated results for key: %@, possible CENs: %@",
-               log: servicesLog, type: .debug, key.cenKey, "\(possibleCENs.count)")
+        os_log("Generated results for key: %@, possible CENs count: %@, CENs: %@", log: servicesLog, type: .debug, key.cenKey,
+               "\(possibleCENs.count)", "\(possibleCENs)")
```

Comment:
@i-schuetz 
There seems to be a problem with how the timestamps are rounded.
In 672 possible cens, during my test, there were only 2 unique values - which makes the whole processing questionable.
We should use CEN lifetime for rounding, not CENKey lifetime.
[possible-cens-problem.txt](https://github.com/Co-Epi/app-ios/files/4458549/possible-cens-problem.txt)

Reply:
That's a bug of the CEN implementation, yes. The rounding should use `CENLifetimeInSeconds`. I guess that it went undetected because we were using 120 seconds as `CENKeyLifetimeInSeconds` (which I changed to 1 week before uploading).